src/components/mostPopular.js

```diff
@@ -12,7 +19,7 @@ const MostPopular = () => {
                         <div className="row">
                             <div className="col-lg-3 col-sm-6">
                                 <div className="item">
-                                    <img src="assets/images/turf1.jpg" alt />
+                                    <img src={turf1} alt />
```

Comment:
please use map when you have to write same content multiple time
for ex- create array of object and map function to itreate it
in future we can have the API.